bazel/repository_locations.bzl

```diff
@@ -137,7 +137,10 @@ REPOSITORY_LOCATIONS = dict(
         strip_prefix = "libevent-1cd8830de27c30c5324c75bfb6012c969c09ca2c",
         urls = ["https://github.com/libevent/libevent/archive/1cd8830de27c30c5324c75bfb6012c969c09ca2c.tar.gz"],
     ),
-    com_github_madler_zlib = dict(
+    net_zlib = dict(
+        # TODO(moderation): revert to com_github_madler_zlib name pending resolution of workaround
```

Comment:
Not sure we need the revert, if `net_zlib` is commonly used in protobuf and/or rules_go, we should just follow it. zlib.net is the canonical URL of the lib per its doc.

Reply:
Make sense. Are we OK with leaving the download target as Github? Seems like the safer / reliable option as I'm not sure where zlib.net is hosted.